pyte/screens.py

```diff
@@ -171,15 +171,11 @@ class Screen(object):
     #: colors.
     default_char = Char(data=" ", fg="default", bg="default")
 
-    #: An infinite sequence of default characters, used for populating
-    #: new lines and columns.
-    default_line = repeat(default_char)
-
     def __init__(self, columns, lines):
         self.savepoints = []
         self.columns = columns
         self.lines = lines
-        self.buffer = []
+        self.buffer = defaultdict(lambda: defaultdict(lambda: Screen.default_char))
```

Comment:
Just use `self.default_char`. There is no point in emphasizing it's "static" imo.